payments/payment-provider.go

```diff
@@ -0,0 +1,99 @@
+package payments
+
+import (
+	"github.com/ethereum/go-ethereum/common"
+	"github.com/statechannels/go-nitro/channel/state"
+)
+
+const alice = 0
+
+type (
+	// A PaymentProvider can be used to make a payment for a given channel.
+	PaymentProvider struct {
+		fp             state.FixedPart
```

Comment:
Is it possible to use `channelId` instead of `state.FixedPart` ?

Reply:
4b9e723c4cf28763d05c253be2a3f4e985163044

Reply:
4b9e723c4cf28763d05c253be2a3f4e985163044